frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -911,6 +915,30 @@ export default function lineAreaBar(
   };
 }
 
+function addValueLabeledChart(
```

Comment:
I initially figured I needed to do something here inside of LineAreaBarRenderer (I basically used Goal) as a guide for "add things to the viz" but ultimately it was confusing about how this and then the post renderer relate to one another. My gut says this can probably just go away but I left it in as a point of discussion.